src/scripts/main.js

```diff
@@ -2,7 +2,27 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard tbody');
 
-// write your code here
+people.forEach(person => {
+  const personInfo = [
+    person.name,
+    (person.sex === 'm') ? 'Male' : 'Female',
+    person.born,
+    person.died,
+    person.died - person.born,
+    Math.ceil(person.died / 100),
+  ];
```

Comment:
it would be better and more convenient to create an object here with the corresponding fields